package.json

```diff
@@ -45,7 +45,7 @@
     "compression": "1.7.4",
     "cors": "2.8.5",
     "deep-extend": "0.6.0",
-    "design-system": "git://github.com/Wikia/design-system.git#22.14.0",
+    "design-system": "git://github.com/Wikia/design-system.git#bdfc0e0ec6cb505f31d01120ea30d78091c57730",
```

Comment:
We should first deploy design-system with version number.

Reply:
So far the process was: we first test on the commit hash and if everything is OK then we release DS and update the version in the clients.

Reply:
The process probably is the same :)

Reply:
Ah, now I get it - so you've approved the changes but wanted to be extra secure so that nobody merges the version with the commit hash after testing? Makes sense!